api/clients/config.go

```diff
@@ -52,9 +52,6 @@ func (c *EigenDAClientConfig) CheckAndSetDefaults() error {
 	if c.ResponseTimeout == 0 {
 		c.ResponseTimeout = 30 * time.Second
 	}
-	if len(c.SignerPrivateKeyHex) != 64 {
```

Comment:
Instead of removing this check, check length only when this value is not nil?

Reply:
SignerPrivateKeyHex is a string defined in the config, it cannot be nil.

Reply:
*empty string

Reply:
https://go.dev/play/p/6gLQTyOMg5o

empty string cannot be compared with nil

Reply:
Oh sorry what I meant was we need to keep this check when the key is non-empty. 
i.e. `if len(c.SignerPrivateKeyHex) > 0 && len(c.SignerPrivateKeyHex) != 64` then return error

Reply:
got it, added